Gir_GstEditingServices.toml

```diff
@@ -0,0 +1,111 @@
+[options]
+girs_dir = "gir-files"
+library = "GES"
+version = "1.0"
+min_cfg_version = "1.14"
+target_path = "gstreamer-editing-services"
+work_mode = "normal"
+# concurrency = "send+sync"
```

Comment:
This should probably be enabled?

Reply:
So everything but the `GstElement` subclasses in GES are not actually thread-safe, so the above is correct but there's still a problem.

First of all, you should move the above configuration to all `GstElement` subclasses (or other `GstObject` subclasses) for consistency.

The actual problem however is that you can add a `GESLayer` to a `GESTimeline` (a `GstElement), and then get that very `GESLayer` out of the `GESTimeline` from another thread. But as `GESLayer` is not thread-safe, this would have to require unsafe code, which it doesn't currently. That's the whole point of the Rust thread-safety guarantees.

Note that this is unrelated of configuring `send+sync` on `GESTimeline`: you could always downcast to `GstElement`, send to a different thread and then upcast to `GESTimeline` again.

I'm not sure what to do about this here, this is a real problem for Rust.